help/guardrails-for-ssm-tagging/index.md

```diff
@@ -0,0 +1,29 @@
+AWS > SSM > Instance Patch Management
+    **Enforce: Enabled with Target Patch Group**
```

Comment:
Add the initial section of the document, pls see the index.md I have given to you, you missed those i.e.

---
layout: help-security
title:  Guardrails for Amazon CloudTrail
slug:   guardrails-for-aws-cloudtrail
date:   2016-01-25 09:00:00
tags:
- Security
- CloudTrail
- AWS
breadcrumbs:
- Security
- Help
---

Reply:
Add a section called Introduction, no need to have any content, in case you want just copy past some generic content from SSM AWS section in introduction section.

Add a section called Steps to the Turbot Policy and then add all these steps you have written.

Pls Pls follow the same standard I have given you as example. Else attaching again in the mail.